test/unit/mdc-chips/mdc-chip.foundation.test.js

```diff
@@ -101,3 +102,22 @@ test('on click, emit custom event', () => {
 
   td.verify(mockAdapter.notifyInteraction());
 });
+
+test('on click in trailing icon, emit custom event', () => {
```

Comment:
You might be able to save some code here by using `test/unit/helpers.captureHandlers`. Example usage within ripple: https://github.com/material-components/material-components-web/blob/master/test/unit/mdc-ripple/foundation-activation.test.js#L27

(You can pass additional properties to the functions it returns, to be assigned to the event object.)

Also, should we actually stub `stopPropagation` and test that it gets called? (I realize right now it needs to exist simply to not throw an error in the first place, but it could also test unintended behavior changes later.)